templates/taxi/driver_detail.html

```diff
@@ -23,4 +23,11 @@ <h4>Cars</h4>
       <p>No cars!</p>
     {% endfor %}
   </div>
+
+  <a href="{% url 'taxi:driver-update-license' pk=driver.id %}" class="btn btn-secondary link-to-page">
+    Update License
+  </a>
```

Comment:
The link to update the driver's license uses the URL name 'driver-update-license'. Ensure this aligns with the intended functionality and does not conflict with the 'driver-update' path in your URL configuration. Clarify the purpose if both URLs are meant to serve different functions.